<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewerUI</class>
 <widget class="QDialog" name="LogViewerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New history view</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="namebox"/>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="devices"/>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLineEdit" name="simpleTimeSpec">
       <property name="text">
        <string>1h</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name of the view:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QCheckBox" name="slidingWindow">
       <property name="text">
        <string>Sliding window</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="simpleTime">
       <property name="text">
        <string>Simple:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Time selection</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="simpleHelpButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="helpButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="extTime">
       <property name="text">
        <string>Extended:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Device(s) or key(s):</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="frombox">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QDateTimeEdit" name="fromdate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="tobox">
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QDateTimeEdit" name="todate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="3" rowspan="2" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>If &quot;To&quot; is not enabled, new values will be added to the plot as they are acquired by the system.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Minimum time (seconds) between two values:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="interval">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="customY">
       <property name="text">
        <string>Custom Y range: from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customYFrom"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customYTo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="15" margin="15"/>
 <tabstops>
  <tabstop>devices</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>simpleTimeSpec</tabstop>
  <tabstop>simpleHelpButton</tabstop>
  <tabstop>simpleTime</tabstop>
  <tabstop>slidingWindow</tabstop>
  <tabstop>extTime</tabstop>
  <tabstop>customY</tabstop>
  <tabstop>customYFrom</tabstop>
  <tabstop>customYTo</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>namebox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LogViewerUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LogViewerUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>devices</sender>
   <signal>returnPressed()</signal>
   <receiver>LogViewerUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
